app/build.gradle

```diff
@@ -57,11 +57,11 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
```

Comment:
I am a bit unsure on the syntax: does this mean we are dropping older android devices or does the app ship with java installed alongside?

Java 17 was released quite recently in terms of old android versions